bndtools.m2e/bnd.bnd

```diff
@@ -33,14 +36,31 @@ Bundle-SymbolicName: bndtools.m2e; singleton:=true
 Bundle-ActivationPolicy: lazy
 
 Import-Package: \
+	org.apache.maven;provider=m2e,\
```

Comment:
As per the shared maven lib, this is a poor way to do this (look how many extra things had to be added!). It should definitely be something more like

    Import-Package: \
        org.apache.maven.*;provider=m2e,\
        org.eclipse.aether.*;provider=m2e,\
        org.codehaus.plexus.*;provider=m2e,\
        *

Reply:
can do!